### ISSUE TITLE
Trackpad Force Touch Acting Up

### ISSUE THREAD

**Issue by emiliobusnelli** [fifafu/BetterTouchTool#2195]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Device information:

Type of Mac:
macOS version: 
BetterTouchTool version: 

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

Hello, fairly new to github and bettertouchtools so please bare with me. I've been having issues with my trackpad force touch on the 2017 Macbook pro. The issue is that my force touch does not respond (has no click/vibration). I restarted my macbook and the force touch began to work again. Once the bettertouchtool opened again my forcetouch was working but soon after it was unresponsive. My trackpad works fine with moving the mouse and tapping on the trackpad, just not the force touch. Has anyone been encountering these same issues? Or would this be an issue with my macbook itself?

Thank you, Emilio.

**Comment by fifafu** [comment]
Which version of BTT  and macOS are you running?

**Comment by emiliobusnelli** [comment]
Type of Mac: MacBook Pro (15-inch, 2017)
macOS version: macOS High Sierra: Version 10.13.3 (17D102)
BetterTouchTool version: 2.340

**Comment by fifafu** [comment]
Does this still happen to you?
(Un)fortunately I haven't had any similar reports yet :-(

**Comment by emiliobusnelli** [comment]
No, this issue has not been happening as of lately. I am unsure why that issue occurred but it seems to have passed. I'm still unsure as to where the problem originated from. Thank you very much for checking in on this issue. I still love BTT to death!
